app/views/schools/core_programme/materials/success.html.erb

```diff
@@ -9,14 +9,14 @@
             <strong><%= @cohort.description %></strong>
           <% end %>
 
-          <p>To access the training materials:</p>
-          <ol class="govuk-list govuk-list--number govuk-!-margin-bottom-7">
-            <li>Visit <%= govuk_link_to 'Support for early career teachers (opens in new tab)',
-            "https://support-for-early-career-teachers.education.gov.uk/users/sign-up",
-            target: :_blank,
-            rel: "noopener noreferrer" %>.</li>
-            <li>Enter your email address to create a new account.</li>
-          </ol>
+        <p class="govuk-body">To access the training materials:</p>
+
+        <p class="govuk-body">
+          Visit <%= govuk_link_to 'Support for early career teachers (opens in new tab)',
+              "https://support-for-early-career-teachers.education.gov.uk",
+              target: "_blank",
+              rel: "noopener noreferrer" %>.
+        </p>
```

Comment:
Think this can be a single sentence now..

```suggestion
        <p class="govuk-body">To access the training materials, visit <%= govuk_link_to 'Support for early career teachers (opens in new tab)',
              "https://support-for-early-career-teachers.education.gov.uk",
              target: "_blank",
              rel: "noopener noreferrer" %>.
        </p>
```